app.js

```diff
@@ -129,29 +188,59 @@ function header(message) {
  * @param {puppeteer.Browser} browser
  * @returns {Promise<boolean>} Are there any drops?
  */
-async function hasValorantDrop(browser) {
+async function hasDrops(browser) {
   header("Dropcheck");
   debug("Opening inventory page ...");
   const page = await browser.newPage();
   await page.goto(`${BASE_URL}/inventory`, { waitUntil: "networkidle2" });
   debug("Querying for drops ...");
-  const noDropsElement = await page.$(NO_DROPS_QUERY);
-  const noDrops = noDropsElement === null;
+  
+  /* Updated by AlexSimpler */
+  let noDrops = await query(page, NO_DROPS_QUERY);
+  //if there are noDrops then length > 0
+  if (noDrops.length) {
```

Comment:
I dislike ints being used as booleans, make it more explicit by doing `noDrops.length !== 0`

Reply:
I think its an efficient way to use them because on cpp ints = bools, but nevermind you can replace it